_includes/manuals/1.17/4.4.2_prov_media.md

```diff
@@ -12,3 +12,10 @@ Be sure to set the Family for the Media
 #### Assign to Operating System
 
 If you have not already done so, return to the Operating System page for your OS and assign the Media to it now.
+
+### Use case
+
+Most sites are well-served with having a 1:1 relationship between operating system and installation media. This will obviously work. However, there is a bunch of use cases that make sense to have more than one set of installation media per operating system. The most common case of this is when a site has datacenters in multiple locations and would like to have the systems automatically select the mirror closed to their location. This can be enforced by only allowing the "correct" location for a given set of installation media.
+
+Another possible use case is to use differnet of installation media for staging of different operating system versions. This is, however significantly more bumpy since this raises a bunch of issues with the version field in the actual operating system and might need other adjustments in the OS deployment process since, for example, the files that are actually used for the tftp boot during network installation are connected to the operating system and not the installation media set. Be prepared to change many templates if you decide doing so.
```

Comment:
s/differnet/different/

Reply:
I'll file a new PR for the obvious one-letter change. Wouldn't it have been easier for all participants to simply merge and then do the one-letter commit?